bitwarden-rs-alpine.json

```diff
@@ -0,0 +1,36 @@
+{
+    "Bitwarden-rs": {
+        "description": "Bitwarden-rs on Alpine",
+        "version": "0.0.1",
+        "website": "https://github.com/dani-garcia/bitwarden_rs",
+        "containers": {
```

Comment:
You could use the `"more_info"` object to add more details on how to setup certs (or a link). You can embed html tags to make it more legible. See the [ZoneMinder Rock-on](https://github.com/rockstor/rockon-registry/blob/bd97e1a93ee28e6a0bf75ad22c5836c02ba0eba8/zoneminder-1.30.json#L37) by @magicalyak  for an example.